Src/BootCamp.Chapter/UserInteraction.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class UserInteraction
+    {
+        private const string characterMask = "\u263A";
+        private const string backspace = "\b \b";
+        private static readonly Credentials newCredentials = new Credentials();
+
+        public static void DisplayMainMenu()
+        {
+            ConsoleInit();
+            DisplayHeader("Main menu");
+
+            Console.WriteLine("(1) Login");
+            Console.WriteLine("(2) Register");
+            Console.WriteLine();
+            Console.WriteLine("(0) Exit!");
+
+            try
+            {
+                ChooseMenu();
+            }
+            catch (InvalidCredentialsDbFile ex)
```

Comment:
Optional: If you know about the catch when syntax totally be cool to use it here :)

Reply:
I know about catch when but first time I use it in a homework Codacy was screaming at me that it does not understand the code.